### ISSUE TITLE
compatibility on o2jam

### ISSUE THREAD

**Issue by jcgo16** [elishacloud/dxwrapper#2]
http://o2jam.com.ph

Theres an issue, when i enable ddrawcompat on the ini it works but the website doesnt work ingame
When i disable ddrawcompat everything lags and the web works again
you can try my game, btw im using the ddrawcompat ddraw on the main game

and another issue with ddrawcompat is you cannot use any overlay apps like discord, twitch, fraps, bandicam

**Comment by elishacloud** [comment]
Hello jcgo16, it looks like these two issues are already reported [here](https://github.com/narzoul/DDrawCompat/issues/9) and [here](https://github.com/narzoul/DDrawCompat/issues/13).

I can try and look at these issues this weekend, if I have time.  But I believe if narzoul has not fixed them then I may not be able to either.  Interestingly enough it seems like o2jam is supposed to support Windows 10.  You should not need to use DDrawCompat for this game at all.

**Comment by jcgo16** [comment]
nope, i just put it like that because we're using ddrawcompat, without it the game will lag to the point the song and the notes are not in sync

**Comment by elishacloud** [comment]
Ok, try this patch.  I made a few changes that should help fix both reported issues:
1. Added an option to prevent DDrawCompat from setting the `setAppCompatData` -> `disableMaxWindowedMode`.
2. Added an option to prevent DDrawCompat from setting the GDI Hook.
3. Added alpha channel support for DDrawCompat to help support overlay apps.
4. Added DxWnd to this package, which is needed to get overlay applications to work.

To install simply drop all 5 files into the o2jam folder, where the main game exe is.

[o2jam.zip](https://github.com/elishacloud/dxwrapper/files/1003129/o2jam.zip)
